### DISCUSSION TITLE
Ask player if they want to buy a pad?

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#646] [Ideas]
When you click on the button to purchase a launch pad it simply does it, and that was never a problem for me before, but I always wondered if the game should ask for confirmation before dropping 20MB on the pad.  Finally this morning I did it: I clicked on it accidentally and bought a pad I didn't mean to.  Not a big deal; a second pad is almost always useful.  But it prompted me to throw out the idea: do others think it would be good to ask for confirmation before proceeding with the purchase?

![AccidentalPurchase](https://user-images.githubusercontent.com/3118014/168483361-70fe8701-1891-48e4-9d7f-faa6c1a2354f.png)

**Comment by rnyoakum** [comment]
I'm going to argue _against_ adding a confirmation window. You're probably the most frequent player of the game, and it has only happened to you once. I don't play often, but have never accidentally bought a pad. I think forcing the player to confirm that they wish to do what they clicked on a button to do is generally bad design, and should only be used when the effects are dramatic or irreversible.

Because the autosave is currently configured to trigger at the start of the turn, if the player misclicked and really doesn't want to spend the money they can reload and repeat their actions. Generally, the hardest part of the turn is planning. They might get different R&D rolls, but otherwise it's straightforward.

**Reply by peyre** [reply]
That makes a lot of sense.  I was hesitant about it in the first place.  Sounds good, I can just leave well enough alone on this one.
